data.sql

```diff
@@ -4,4 +4,14 @@ INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
     ('Agumon', '2020-02-03', 0, true, 10.23),
     ('Gabumon', '2018-11-15', 2, true, 8),
     ('Pikachu', '2021-01-07', 1, false, 15.04),
-    ('Devimon', '2017-05-12', 5, true, 11);
\ No newline at end of file
+    ('Devimon', '2017-05-12', 5, true, 11);
+
+
+    INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) values
+    ('Charmander', '2020-02-08', 0, false, -11),
+    ('Plantmon', '2021-11-15', 2, true, -5.7),
+    ('Squirtle', '1993-04-02', 3, false, -12.13),
+    ('Angemon', '2005-06-12', 1, true, -45),
+    ('Boarmon', '2005-06-07', 7, true, 20),
```

Comment:
* Kindly add the correct weight for animal name( Boarmon) `20.4kg` 
![image](https://github.com/albertkantwi/Vet_clinic/assets/52497246/835967af-6121-430e-877b-621a0a0480ee)